### DISCUSSION TITLE
Do i need to use MultiTenantIdentityDbContext ?

### DISCUSSION THREAD

**Discussion by rushangp1** [Finbuckle/Finbuckle.MultiTenant#675] [Q&A]
I'm using Database Per Tenant, Do i need use MultiTenantIdentityDbContext specifically for my master database?

**Comment by AndrewTriesToCode** [comment]
No you don’t. Just a way to get the correct correction string per tenant so maybe injecting a TenantInfo into your own context or via some other means.
